Entity/Player/Player.gd

```diff
@@ -2,11 +2,50 @@ extends Entity
 class_name Player
 ## Base player Entity class.
 
+var _should_save_persistent_data: bool = true
```

Comment:
I only see usage of this variable in the tests. Will this be used outside of testing?

Reply:
It'll probably be useful when implementing the external storage (not the one between fights), if I understood Cheesy's goal correctly

Reply:
I'm not sure if it will be used for saving actually. We just don't want to have persistence with the player when testing because when a new Player is spawned in a new test then the data will carry over

Reply:
Okays. Just checkin 👍🏽